packages/contracts/src/connect-contracts.ts

```diff
@@ -63,13 +63,13 @@ export const connectL1Contracts = async (
   return {
     addressManager: getEthersContract('Lib_AddressManager'),
     canonicalTransactionChain: getEthersContract(
```

Comment:
Same as above

Reply:
So where does this come from then if the linting workflow step passed? And how do I test locally and auto fix?

Reply:
Good question. I will look into it.

Reply:
Running `yarn lint:fix` fixes the issue. I'm not sure why the lint workflow isn't catching this, I need to look into it.